tests/onnx/data/reference_graphs/original_nncf_graph/vgg16.dot

```diff
@@ -1,81 +1,101 @@
 strict digraph  {
-"0 Conv_0" [id=0, type=Conv];
```

Comment:
Consider expanding test cases to cover both training-mode and eval-mode input models. Separate the reference files accordingly.

Reply:
I don't think that we should expand model covering by adding train mode as OpenVINOExecutionProvider doesn't support inference of some layers exported in training mode, e.g. BatchNorm